middlewares/parsed-cookies.middleware.js

```diff
@@ -0,0 +1,4 @@
+export default (req, res, next) => {
+    req.parsedCookies = req.cookies;
```

Comment:
Consider using "cookie" npm package as a more generic way of adding parsed cookies in the request.
Also take into account that cookies are mostly passed in headers.